strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/BrowseController.java

```diff
@@ -145,26 +151,17 @@
             
             String matchedPattern = (String) request.getAttribute(HandlerMapping.BEST_MATCHING_PATTERN_ATTRIBUTE);
             String subPath = new AntPathMatcher().extractPathWithinPattern(matchedPattern, request.getRequestURI());
-            Path baseDir = Paths.get(repository.getBasedir().toString(), subPath);
+            Path baseDir = Paths.get(repository.getBasedir().toString(), subPath.replaceAll("/", Matcher.quoteReplacement(File.separator)));
```

Comment:
you should resolve paths with `RepositoryProvider.resolvePath` method

Reply:
+1